v2022.01.21-0 release notes (#662)
replicatedhq/kurl.sh: src/release-notes/v2022.01.21-0.md

---
date: "2022-01-21"
version: "v2022.01.21-0"
weight: 202201210
---

### <span class="label label-blue">Improvements</span>
- Adds [KOTS add-on](/docs/add-ons/kotsadm) version v1.59.3.
- Adds [Prometheus add-on](/docs/add-ons/prometheus) version 0.53.1-30.1.0.
- Adds [EKCO add-on](/docs/add-ons/ekco) version 0.15.0, which supports auto-renewal of Contour and Envoy certs.

> - Moves the [`latest`](https://kurl.sh/latest) installer on kurl.sh moves to Kubernetes 1.22.5.

Reviewer: ```suggestion
- Moves the [`latest`](https://kurl.sh/latest) installer on kurl.sh to Kubernetes 1.22.5.
```

### <span class="label label-orange">Bug Fixes</span>

> - Fixes a bug that caused the "Internal Storage" snapshot option to be missing when an object store is available.

Reviewer: ```suggestion
- Fixes a bug that caused the **Internal Storage** snapshot option to be missing when an object store is available.
```

---
date: "2022-01-21"
version: "v2022.01.21-0"
weight: 202201210
---

### <span class="label label-blue">Improvements</span>
- Adds [KOTS add-on](/docs/add-ons/kotsadm) version v1.59.3.
- Adds [Prometheus add-on](/docs/add-ons/prometheus) version 0.53.1-30.1.0.
- Adds [EKCO add-on](/docs/add-ons/ekco) version 0.15.0, which supports auto-renewal of Contour and Envoy certs.
- Moves the [`latest`](https://kurl.sh/latest) installer on kurl.sh to Kubernetes 1.22.5.

### <span class="label label-orange">Bug Fixes</span>
- Fixes a bug that caused the **Internal Storage** snapshot option to be missing when an object store is available.

> - Fixes random Alert Manager and Grafana Nodeports in the Prometheus addon for versions 0.53.1-30.1.0+.

Reviewer: ```suggestion
- Fixes random Alert Manager and Grafana Nodeports in the Prometheus add-on for versions 0.53.1-30.1.0+.
```
